### ISSUE TITLE
Regular unexpected 403 errors in the logs

### ISSUE THREAD

**Issue by kumy** [geokrety/geokrety-website#852]
I see regularly such error in the logs. Seems related to the PHP sessions. Retrying works.

I need to investigate it.
```
 NOTICE: PHP message: HTTP 403 (GET /api/v1/export2?wpt=GCAE26Q)
 NOTICE: PHP message: [/var/www/geokrety/vendor/bcosca/fatfree-core/db/sql/session.php:77] Base->error()
 NOTICE: PHP message: [geokrety.php:101] Base->exists()
```

**Comment by kumy** [comment]
Seems related to Session hijacking detection in FatFreeFramework

Code excerpt
```
		if ($this->get('ip')!=$this->_ip || $this->get('agent')!=$this->_agent) {
			$fw=\Base::instance();
			if (!isset($this->onsuspect) ||
				$fw->call($this->onsuspect,[$this,$id])===FALSE) {
				//NB: `session_destroy` can't be called at that stage (`session_start` not completed)
				$this->destroy($id);
				$this->close();
				unset($fw->{'COOKIE.'.session_name()});
				$fw->error(403);
			}
		}
```

**Comment by kumy** [comment]
I applied a non-persistent small workaround, so user experience will not be impacted. But this really need proper analysis...

**Comment by kumy** [comment]
Fixed by 47eb7a3cb452949b8290b5da4bcab036afc7b6af
